features/breol_news/breol_news.install

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the breol_news module.
+ */
+
+/**
+ * Move news content from body to lead.
+ */
+function breol_news_update_7101() {
+  $nodes = node_load_multiple([], ['type' => 'breol_news']);
```

Comment:
How many items are we migration here? Should the batch API have been used and what about the format of the content moved?

![Screenshot 2021-04-13 at 15 13 44](https://user-images.githubusercontent.com/111397/114558357-e2c59c80-9c6a-11eb-82bd-1538bcc86f27.png)

Reply:
We're talking about around 500 nodes. The (text) format on the old and the new field is the same.